src/NovelRT.Interop/Audio/NrtAudioService.cpp

```diff
@@ -105,7 +169,16 @@ NrtResult Nrt_AudioService_stopMusic(NrtAudioService service) {
     }
 
     auto serv = reinterpret_cast<Audio::AudioService*>(service);
-    serv->stopMusic();
+
+    try
+    {
+      serv->stopMusic();
+    }
+    catch (const Exceptions::NotInitialisedException)
+    { // todo: handle error message
```

Comment:
This looks like a custom exception. Do we have an internal function for the error message?